_data/simple-icons.json

```
@@ -5225,6 +5225,21 @@
             "source": "https://gradle.com/brand",
             "guidelines": "https://gradle.com/brand"
         },
+        {
+            "title": "Gradle Play Publisher",
+            "hex": "82B816",
+            "source": "https://github.com/Triple-T/gradle-play-publisher/blob/df4eadf1ca6b5bad50e21be0b21816722ed50342/assets/logo.svg",
+            "license": {
+                "type": "MIT",
+                "url": "https://github.com/Triple-T/gradle-play-publisher/blob/df4eadf1ca6b5bad50e21be0b21816722ed50342/LICENSE"
```

```suggestion
                "type": "MIT"
```

Assuming it's a 'vanilla' MIT license (doesn't have any caveats) then we can remove the URL here!